docker/config/README

```diff
@@ -0,0 +1,9 @@
+To use the gui tool will need xhost on mac 
```

Comment:
Consider saving this as a markdown (.md) file and formatting it.

Reply:
Changed README to a markdown file and formatted